src/Tizen.NUI/src/public/BaseComponents/ControlState.cs

```diff
@@ -86,20 +84,24 @@ public class ControlState : IEquatable<ControlState>
         /// This is used in a selector class. It represents all other states except for states that are already defined in a selector.
         /// </summary>
         /// <since_tizen> 9 </since_tizen>
-        public static readonly ControlState Other = Create("Other");
+        public static readonly ControlState Other = new ControlState(nameof(Other));
 
-        private List<ControlState> stateList = new List<ControlState>();
-        private readonly string name = "";
+        readonly ulong bitFlags;
 
         /// <summary>
         /// Gets or sets a value indicating whether it has combined states.
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public bool IsCombined => stateList.Count > 1;
+        public bool IsCombined => bitFlags != 0UL && Math.Ceiling(Math.Log2(bitFlags)) != Math.Floor(Math.Log2(bitFlags));
 
-        private ControlState() { }
+        ControlState(ulong bitMask)
```

Comment:
It'd be better to have explicit `private` keyword like other methods in this file.